src/ZipkinBundle/Components/Messenger/ZipkinSendHandler.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+
+namespace ZipkinBundle\Components\Messenger;
+
+use Symfony\Component\Messenger\Envelope;
+
+use Zipkin\Kind;
+use Zipkin\Propagation\B3;
+use Zipkin\Tags;
+use Zipkin\Tracing;
+
+class ZipkinSendHandler implements ZipkinHandlerInterface
+{
+    /**
+     * @var \Zipkin\Tracer
+     */
+    private $tracer;
+
+    public function __construct(Tracing $tracing)
```

Comment:
I am curious about this receiving tracing instead of tracer. Couldn't we just receive the tracer? Is it because of the DI? if that is the case I think we somewhere explain how to create the `tracer`. Maybe we need to make it explicit for avoiding this cases?

Reply:
I followed the same configuration of `ZipkinBundle\Middleware`. In this way, we also leave the possibility to create custom Tracing

Reply:
I see, the main thing here is that Tracing contains more things than we need so probably better to declare a tracer in the bundle and use it instead of the tracing. What do you think?

Reply:
Ok, I like it!

Reply:
@jcchavezs , just for reference in brave we pass tracing to the instrumentations; but this might change between languages.